app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
   has_secure_password
+  scope :ordered, -> { order(:created_at) }
```

Comment:
```suggestion
  scope :ordered, -> { order :created_at }
```
cơ mà scope viết sau callback và validate nhé